programs/solana/programs/ics07-tendermint/src/helpers.rs

```diff
@@ -24,17 +24,17 @@ pub fn deserialize_misbehaviour(bytes: &[u8]) -> Result<Misbehaviour> {
 
 pub fn validate_proof_params(
     client_state: &Account<ClientState>,
-    msg: &MembershipMsg,
+    msg: &impl ICS25Msg,
 ) -> Result<()> {
     require!(!client_state.is_frozen(), ErrorCode::ClientFrozen);
 
     require!(
-        msg.height <= client_state.latest_height.revision_height,
+        msg.height() <= client_state.latest_height.revision_height,
         ErrorCode::InvalidHeight
     );
```

Comment:
No need to check this, everything will fail when we try to validate the consensus state right?